mod-patron-blocks/src/main/resources/domain/mod-patron-blocks/features/util/messages-text.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Shouldn't these messages be stored in `patron-block-conditions`?

Reply:
They are rewritten after each put request, so we have to save them also. We don't have separate file for patron block conditions put requests, so we store the messages in this file